### PR TITLE
TMDM-10164 remove duplicated 'nodeToString' and 'styleDocument' method.

### PR DESCRIPTION
**Pull request by yjli-talend**

### CODE REVIEW

File: main/plugins/org.talend.mdm.commmon/src/main/java/org/talend/mdm/commmon/util/core/XmlUtil.java [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+import org.dom4j.io.DocumentSource;
+import org.w3c.dom.Node;
+
+public class XmlUtil {
+
+    private static final Logger logger = Logger.getLogger(XmlUtil.class);
+
+    private static final TransformerFactory transformerFactory_Xalan = new org.apache.xalan.processor.TransformerFactoryImpl();
+
+    private static final TransformerFactory transformerFactory_Saxon = new net.sf.saxon.TransformerFactoryImpl();
```
Comment (glzhou) [inline main/plugins/org.talend.mdm.commmon/src/main/java/org/talend/mdm/commmon/util/core/XmlUtil.java]: underscore is not a Java variable naming convention, use CamelCase instead

**Comment by glzhou** [comment]
Looks OK to me.

**Comment by pbabin** [comment]
What about setting the property via the JVM parameter?  Service mechanism is correct but faces some concerns:  first it needs to go through all the classpath for discovery and second it relies on order in the classpath this is then quite difficult to maintain.

**Comment by glzhou** [comment]
as per https://github.com/Talend/tmdm-server-ee/pull/42/commits/39b33fcbb37cacab7e6b574c6a502a6ed26ca974, I think it's good to merge

**Pull request merged**